### DISCUSSION TITLE
Redirects to other domains and "Failed to fetch RSC payload"

### DISCUSSION THREAD

**Discussion by elie222** [vercel/next.js#61841] [Help]
### Summary

I use Next.js redirects as a URL shortener.

Eg. https://getinboxzero.com/github points to the Inbox Zero GitHub.

But then when a user loads the page and that link appears in the page, I get an error: "Failed to fetch RSC payload for https://www.getinboxzero.com/github"

Any idea how to disable prefetching? I tried to disabled this on `next/link` component but didn't have an effect.

### Example

https://github.com/elie222/inbox-zero/blob/main/apps/web/next.config.mjs

**Comment by zudsniper** [comment]
![image](https://github.com/user-attachments/assets/fa3d66ba-7039-4d6b-99ee-ad5ca64c242a)
Not sure to what extent this is helpful, but I am self-hosting behind a cloudflared tunnel, and I'm encountering this QUIC error... that, curiously, ends up with a 200 OK... or so it seems? 

Apologies if this isn't related, it just seems like it could be (from my limited perspective)
